examples/README.md

```diff
@@ -0,0 +1,45 @@
+# Examples
+
+When you add a new sampling fuctions, please add an Exmaple Notebook to describe how the function is used. Before push your function and its corresponding notebook to `Main` branch, please convert the notebook into text file.
+
+## Converting Notebook Files to Text Files
```

Comment:
Could we add a few sentences on how to edit existing examples?

Reply:
Sure. Can we do something like this, what do you think?

## Converting Notebook Files to Text Files

Please convert your `.ipynb` to `markdown` before commit your example to this repo. You can use the command below:

```python
jupytext example/your_example_notebook.ipynb --to myst
```

Once the example file is converted to a Markdown file, you have two options for editing:

1. Edit the Markdown version as it is a regular Markdown file.
2. Edit the Notebook version and convert it to a Markdown file with the command above. `Jupytext` can handle the change if the example has the same file name.

The script will produce the `Markdown` version of your example notebook. Please make sure you add only the `Markdown` version to your commit.

Reply:
LG overall - BTW is there a way to convert the .md file back to .ipynb?

Reply:
`jupytext examples/example_file.md --to notebook` should do it.

Reply:
Nice - please go ahead and add these into the README

Reply:
Got it. Added and pushed.